### ISSUE TITLE
[bug]: Mega menu is not accessible in keyboard tab order

### ISSUE THREAD

**Issue by dani97** [magento/pwa-studio#3384]
**Describe the bug**

Mega menu sub categories are not accessible.

**To reproduce**

1. Open PWA home page
2. move to mega menu item with keyboard tab order
3. Press down, enter, space, arrow keys to access sub menu
4. You would not be able to interact with sub categories.

**Expected behavior**

mega menu should implement https://www.w3.org/TR/wai-aria-practices/#menu

**Please complete the following device information:**

-   Device [ PC]:
-   Browser [e.g. Chrome, Safari]:
-   Browser Version [e.g. 22]:
-   Magento Version [e.g Magento Commerce 2.4]:

**Please let us know what packages this bug is in regards to:**

-   [ ] `venia-concept`
-   [x] `venia-ui`
-   [ ] `pwa-buildpack`
-   [ ] `peregrine`
-   [ ] `pwa-devdocs`
-   [ ] `upward-js`
-   [ ] `upward-spec`
-   [ ] `create-pwa`

**Comment by huykon** [comment]
@dani97 I checked site https://develop.pwa-venia.com/ and follow your steps but the menu working well. It's not like your report

**Comment by dani97** [comment]
@huykon only down arrow interaction works, but we have to implement it like this https://www.w3.org/TR/wai-aria-practices/examples/menubar/menubar-2/menubar-2.html#code-ex-1. Also we have to provide role attribute for submenu and menu items for screen reader to identify the submenus.

**Comment by glo42707** [comment]
@dani97 

as per retesting by QA team issue is not reproducible kindly please help us here with more information if we need to do any specific configurations to reproduce.

**Comment by dani97** [comment]
@glo42707 I don't remember what I exactly faced at that time. Please feel free to close it if not reproduceable. Thanks

**Comment by glo82145** [comment]
As this issue is not reproducible at our end, hence we are closing this issue.
